tensorflow_datasets/image/food101.py

```diff
@@ -25,7 +25,7 @@
 import tensorflow.compat.v2 as tf
 import tensorflow_datasets.public_api as tfds
 
-_BASE_URL = "http://data.vision.ee.ethz.ch/cvl/food-101.tar.gz"
+_BASE_URL = "https://data.vision.ee.ethz.ch/cvl/food-101.tar.gz"
```

Comment:
If you modify the urls, could you re-generate the checksums files with `--register_checksums` ?
https://www.tensorflow.org/datasets/add_dataset#2_run_download_and_prepare_locally